docs/05-icons.md

```diff
@@ -2,6 +2,7 @@
 title: Icons
 ---
 ### Adding a new icon
+- Use SVGOMG to optimize the SVG https://jakearchibald.github.io/svgomg/
```

Comment:
It will also be done automatically by the build step so not critical. (But it doens't hurt :))